app/forms/contact_email_form.rb

```diff
@@ -6,7 +6,7 @@ class ContactEmailForm < ApplicationForm
   validates :email, presence: true, on: :deposit
   validates :email, format: {
     with: URI::MailTo::EMAIL_REGEXP,
-    allow_blank: true,
+    allow_blank: false,
```

Comment:
This is going to require an email for a draft. Email should be optional for a draft.